app/controllers/concerns/idv_session.rb

```diff
@@ -1,6 +1,10 @@
 module IdvSession
   extend ActiveSupport::Concern
 
+  included do
+    before_action :sp_context_needed?
```

Comment:
as long as we're here, can we rename this to be some sort of statement? Since it's a side-effecty method and not a simple predicate? Like `redirect_if_sp_context_needed` or `require_sp_context`